tests/test_addr.py

```diff
@@ -73,9 +80,14 @@ def test_destroy_ah(self):
                     continue
                 gr = get_global_route(ctx)
                 ah_attr = AHAttr(gr=gr, is_global=1, port_num=port_num)
-                with AH(pd, attr=ah_attr) as ah:
-                    ah.close()
-                    done += 1
+                try:
+                    with AH(pd, attr=ah_attr) as ah:
+                        ah.close()
+                        done += 1
+                except PyverbsRDMAError as ex:
+                    if 'create' in str(ex) and 'not implemented' in str(ex):
+                        raise unittest.SkipTest('Create AH is not supported')
```

Comment:
Ditto - wording. Please keep the consistent style and past sentence like existing sentences.

Reply:
> Please keep the consistent style and past sentence **like existing sentences**.
Where have you seen this (past sentence) style in the existing messages ? 
I don't think it's appropriate here to use past sentence...